scripts/dev/clang-tidy.sh

```diff
@@ -16,9 +16,11 @@ fi
 
 source_root=$(realpath "$(dirname "$(dirname "$(dirname "$0")")")")
 
-files=$(find "$source_root/include" "$source_root/cli" "$source_root/samples" \
-    "$source_root/lib/error" "$source_root/lib/logging" \
-    "$source_root/lib/rest" -name "*.[ch]" -type f -print0 | xargs --null)
+files=$(find "$source_root/cli" "$source_root/include" \
+    "$source_root/lib/config" "$source_root/lib/error" \
+    "$source_root/lib/logging" "$source_root/lib/pidfile" \
+    "$source_root/lib/rest" "$source_root/samples" \
+    -name "*.[ch]" -type f -print0 | xargs --null)
```

Comment:
No point in using `-print0` if you just put the output back into a string variable.  Why even bother with the local var?  You can do this:
```
find <args> -print0 | xargs -r --null clang-tidy <args>
```
That assumes `clang-tidy` exists with error status when it should.

Reply:
I am not a bash god like you 😄. I will update it.